string ModulName
---
bool Erfolg
string SystemweiterModulName
int16 ModuleNumber